### PR TITLE
makes synthfesh heal burn husks when they are under 50 damage and there is over 100 units of synthflesh present

### PR DESCRIPTION
**Pull request by SomeguyManperson**
# General Documentation

### Intent of your Pull Request

make synthflesh more viable by giving it more utility
also needing a high tier surgery to just unhusk someone is a pain in the ass

### Why is this change good for the game?
make medbay less pain and less reliant on clooning
# Wiki Documentation

### Briefly describe your PR and the impacts of it, in layman's terms. 
synthflesh can now heal burn husks specifically if the total volume reaches 100 while  burn damage is under 50

### What should players be aware of when it comes to the changes your PR is implementing?
synthflesh can now unhusk people if the husking is solely from burns
### What general grouping does this PR fall under? 
medbay

### If there are any numerical values involved in your PR that will be relevant to a player, please note them here. 
Synthflesh will unhusk burn-related husk corpses when:
the corpse has less than 50 burn damage
synthflesh is dosed to at least 100 units
unhusking will ONLY occur on synthflesh being applied so make sure to do that part first

# Changelog

:cl:  
tweak: synthflesh will now heal burn husks if their burn damage is under 50 and the total amount of synthflesh dosed reaches 100
/:cl:

### CODE REVIEW

**Comment by Xoxeyos** [comment]
The Heretic Flesh path may be unbalanced by this, one of their abilities is to make a slave by killing them, and then using the transmutation rune with them over it, husking them. I don't believe heretics should be able to run around with two 100~ health slaves. [
image
](url)

**Comment by SomeguyManperson** [comment]
this should only reverse husking from burns, not from other sources but I'll have to double check that

**Comment by Xoxeyos** [comment]
> 
> 
> this should only reverse husking from burns, not from other sources but I'll have to double check that

Please do tell me when you have tested this on a heretic ghoul, and if you fix any problems with it, I'll change my vote.

**Comment by SomeguyManperson** [comment]
heretic ghouling uses generic husking which isn't cured by specific burn dehusking

**Review by JamieD1 — APPROVED** [review]
